packages/portfolio-contract/src/portfolio.flows.ts

```diff
@@ -770,3 +766,9 @@ export const openPortfolio = (async (
   /* c8 ignore end */
 }) satisfies OrchestrationFlow;
 harden(openPortfolio);
+
+export const makeLCA = (async (orch: Orchestrator) => {
+  const agoricChain = await orch.getChain('agoric');
+  const lca = await agoricChain.makeAccount();
+  return lca;
```

Comment:
curious why await?

```suggestion
  return agoricChain.makeAccount();
```

Consider defining the return type of the function

Reply:
No particularly good reasons (I'll clean it up), but a couple possible motivations for this sort of await come to mind:

 - to log the resolution of the promise and/or look at it in a debugger
 - to include `makeLCA` in the stack trace if the promise from `makeAccount()` rejects